addon/templates/components/editable-field.hbs

```diff
@@ -1,11 +1,14 @@
 <span class="content">
   {{#unless isEditing}}
-    <span onclick={{perform edit}} class="clickable editable">
+    <span class="clickable editable">
       {{#if value}}
         {{#if looksEmpty}}
           {{fa-icon "edit" class="enabled"}}
```

Comment:
This also needs to respond to click `{{fa-icon "edit" class="enabled" click=(perform edit}}`.

Reply:
Added.